libs/wingc/src/jsify.rs

```diff
@@ -1434,7 +1435,16 @@ impl<'a> FieldReferenceVisitor<'a> {
 							.as_variable()
 							.unwrap(),
 					)
-				} else if let Some(s) = object.evaluated_type.borrow().unwrap().as_struct() {
+				} else if let Some(iface) = obj_type.as_interface() {
+					Some(
+						iface
+							.env
+							.lookup(&property, None)
+							.expect("covered by type checking")
+							.as_variable()
+							.unwrap(),
+					)
+				} else if let Some(s) = obj_type.as_struct() {
```

Comment:
This was needed to fix a separate bug in the `Doubler` example where the compiler wouldn't allow `this.func` to be used inflight because it's an interface.